src/Microsoft.ML.Tokenizers/Model/EnglishRoberta.cs

```diff
@@ -470,9 +470,9 @@ private IReadOnlyList<Token> ModifyTokenListOffsets(IReadOnlyList<Token> tokens,
             return vocab;
         }
 
-        private Dictionary<(string, string), int> GetMergeRanks(Stream mergeStream)
+        private Cache<(string, string), int> GetMergeRanks(Stream mergeStream)
         {
-            var mergeRanks = new Dictionary<(string, string), int>();
+            var mergeRanks = new Cache<(string, string), int>(60_000);
```

Comment:
Where does this 60k come from?

Reply:
The loaded data from the merge file is 50K. I give it 10K more to grow.